runtime/bindings/python/src/pyopenvino/graph/function.cpp

```diff
@@ -589,6 +589,7 @@ void regclass_graph_Function(py::module m) {
         "add_outputs",
         [](ov::Function& self, py::handle& outputs) {
             int i = 0;
+            std::vector<ov::Output<ov::Node>> new_outputs;
```

Comment:
looks like it should be `ov::Output<const ov::Node>`, that should fix python tests